README.rst

```diff
@@ -225,10 +229,14 @@ Visual Studio Code
      $ where darker
      %LocalAppData%\Programs\Python\Python36-32\Scripts\darker.exe  # possible location
 
-3. Add these configuration options::
+3. Add these configuration options to VS code, ``Cmd-Shift-P``, ``Open Settings (JSON)``::
+   
 
     "python.formatting.provider": "black",
-    "python.formatting.blackPath": "<install_location_from_step_2>"
+    "python.formatting.blackPath": "<install_location_from_step_2>",
+    "python.formatting.blackArgs": ["--diff"],
+
+Extra argument can be passed to ``darker`` along side, but ``--diff`` is necessary.
```

Comment:
The meaning of this wasn't immediately obvious to me after reading it once. Would this wording with an example included be clear:
```suggestion
You can pass additional arguments to ``darker`` in the ``blackArgs`` option (e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
```

Reply:
sounds good.